server/libs/infra/src/infra.config.ts

```diff
@@ -1,17 +1,20 @@
 import { QueueName } from '@app/domain';
+import { Logger } from '@nestjs/common';
 import { BullModuleOptions } from '@nestjs/bull';
 import { RedisOptions } from 'ioredis';
 import { InitOptions } from 'local-reverse-geocoder';
 import { ConfigurationOptions } from 'typesense/lib/Typesense/Configuration';
 
+const logger = new Logger('infra.config');
+
 function parseRedisConfig(): RedisOptions {
   const redisUrl = process.env.REDIS_URL;
   if (redisUrl && redisUrl.startsWith('ioredis://')) {
     try {
       const decodedString = Buffer.from(redisUrl.slice(10), 'base64').toString();
       return JSON.parse(decodedString);
     } catch (error) {
-      throw new Error(`Failed to decode redis options: ${error}`);
+      logger.error(`Failed to decode redis options: ${error}`);
```

Comment:
This one should probably throw an error to prevent the app from starting if the config is invalid...